script/Arcueid/Day 5/05_00_ARC05_8.txt

```diff
@@ -88,7 +88,7 @@ The white beauty in front of him is swallowed completely.
 [sha:9c453a2810079dd648098657d6d38fc360da3040]{
 -- Page 522, Offset 9997.
 -- 「――――――、え？」
-"――――――, Eh?"
+"――――――Eh?"
```

Comment:
We can consider changing the `Eh`s to `Huh`s to appease the nitpickers who don't understand that Shiki is just Canadian

Reply:
I can do that if you want. We already localized the fufufus anyway